docs/STOLEN.md

```diff
@@ -0,0 +1,30 @@
+# Stolen
+
+Tips how to prevent and/or minimize possible abuse of your Microsoft account.
+This applies only to accounts stored or managed by IAS, not third-party or official game launchers, mods, or other apps.
+You can learn how to protect your account within IAS by looking at the [Crypt](CRYPT.md) page.
+
+## Aftermath
+
+If you think your account is ALREADY stolen from IAS, you SHOULD do this:
+
+1. Immediately delete all your accounts from the IAS mod AND from any other launchers, apps and/or mods.
+2. Close the game.
+3. Go to [account.live.com/password/Change](https://account.live.com/password/Change) and change your Microsoft account password.
+4. Go to [account.microsoft.com/consent/manage](https://account.live.com/consent/manage) and revoke any apps/mods/launchers/etc. related to Minecraft (such as In-Game Account Switcher, MultiMC, Prism Launcher, Modrinth and/or any others) AND any apps that you don't recognize OR don't use.
```

Comment:
The text here says it'll lead to microsoft.com but the link is to the live.com domain